client/app/services/query.js

```diff
@@ -184,6 +186,10 @@ export class Query {
     return this.$parameters;
   }
 
+  applyAutoLimit() {
+    return this.options.applyAutoLimit;
+  }
+
```

Comment:
I'd prefer to access directly from `this.options.applyAutoLimit` in the methods above as it's only used in this class anyway.

Could also rename it or make it a getter, `applyAutoLimit` implies that it will execute something to apply the auto limit 😬.